noxfile.py

```diff
@@ -305,6 +305,7 @@ def docs(session: nox.Session) -> None:
     if "nightly" in session.posargs and "internal" in session.posargs:
         rustdoc_flags.append("--Z unstable-options")
```

Comment:
If this removes all usage of the `docsrs` flag, maybe we want to remove it form the Noxfile here as well?

Reply:
Unfortunately there's a few uses remaining, related to the `doc_cfg` feature.